src/Azure.Functions.Cli/Diagnostics/ColoredConsoleLogger.cs

```diff
@@ -88,11 +121,24 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             LogToConsole(logLevel, exception, formattedMessage);
         }
 
-        private void LogToConsole(LogLevel logLevel, Exception exception, string formattedMessage)
+        private void LogToConsole(LogLevel logLevel, Exception exception, string formattedMessage, bool includeTimeStamp = true)
         {
             foreach (var line in GetMessageString(logLevel, formattedMessage, exception))
             {
-                ColoredConsole.WriteLine($"[{DateTime.UtcNow.ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffZ", CultureInfo.InvariantCulture)}] ".DarkGray() + line);
+                string prefix = includeTimeStamp
+                    ? $"[{DateTime.UtcNow.ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffZ", CultureInfo.InvariantCulture)}] ".DarkGray().ToString()
+                    : string.Empty;
+
+                ColoredConsole.WriteLine(prefix + line);
+            }
+        }
+
+        private void LogJsonToFile(string message)
```

Comment:
To be fully sure we are on the same page, the file would not be a valid JSON but rather contain JSON every line?
Example --
```
{ "a":"b"}
{"d":"e"}
```

Reply:
That is correct. See https://jsonlines.org/ for some examples